Gopher-Industries/Team-TaskforceX
# 210-Fonts-Colours-Search-Dimensions

Summary of changes:

-Added active material search bar.
-Changed fonts and weights (waiting on Ashley's push to confirm). 
-Changed heights and radii of chips and buttons. 
-Changed margins to bring the screen in-line with wireframe (shifted entire screen down, arrow and continue button) 
-Changed colours to match wireframe, including background. 
-Added additional comments.
-Cleaned up code.
-Confirmed inputs still carry across to preferences. 
-Confirmed back arrow functions.

Testing Instructions:

-May need to "npm i @expo-google-fonts/open-sans"
-Confirm code runs appropriately.
-Confirm active search bar (Inputs from list are not added to "Added by you").
-Confirm inputs are carrying over to "Preferences".

![HealthConditions 1](https://user-images.githubusercontent.com/118961659/207214719-a3ff4ecc-2e1e-4ef6-bb34-84f2958c8469.PNG)
![HealthConditions 2](https://user-images.githubusercontent.com/118961659/207214723-8ad8a326-3830-448c-b5fa-b0baf8cd9dac.PNG)
![HealthConditions 3](https://user-images.githubusercontent.com/118961659/207214725-30358ea1-9b5e-416d-948a-28c1885510ba.PNG)
